job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/Job.groovy

```diff
@@ -617,6 +617,39 @@ abstract class Job extends Item {
         }
     }
 
+    /**
+     * Enables job rebuild functionality
+     *
+     * @since 1.39
+     */
+    @RequiresPlugin(id = 'rebuild', minimumVersion = '1.25')
+    void enableJobRebuild(boolean automaticallyUseParmetersFromPreviousRun = false) {
+        withXmlActions << WithXmlAction.create { Node project ->
+            project / 'properties' / 'com.sonyericsson.rebuild.RebuildSettings' {
+                autoRebuild automaticallyUseParmetersFromPreviousRun
+                rebuildDisabled false
+            }
+        }
+    }
+
+    /**
+     * Disables job rebuild functionality
+     *
+     * @since 1.39
+     */
+    @RequiresPlugin(id = 'rebuild', minimumVersion = '1.25')
```

Comment:
The DSL does not provide separate methods to enable/disable features. The DSL should look like this instead:

```groovy
job {
    properties {
        rebuild {
            autoRebuild(boolean autoRebuild = true)
            rebuildDisabled(boolean rebuildDisabled = true)
        }
    }
}
```